processModels/summary/recap/q-essay-sequentialIterative.md

```diff
@@ -13,30 +13,30 @@ d) Customer satisfaction.
 
 e) Monitoring the project progress.
 
-f) Suitability for a school project
+f) Suitability for a school project.
 
 <div slot="answer">
 
 a) Quality of the final product:
 
 * Iterative: Frequent reworking can deteriorate the design. Frequent refactoring should be used to prevent this. Frequent customer feedback can help to improve the quality (i.e. quality as seen by the customer).
-*	Sequential: Final quality depends on the quality of each phase. Any quality problem in any phase could result in a low quality product.
+*	Sequential: Final quality depends on the quality of each phase. Any quality problems in any phase could result in a low quality product.
 
 b) Risk of overshooting the deadline.
 
-* Iterative: Less risk. If the last iteration got delayed, we can always deliver the previous version. However, this does not guarantee that all features promised at the beginning will be delivered on the deadline.
+* Iterative: Less risk. If the last iteration got delayed, we can always deliver the previous version. However, this does not guarantee that all features promised at the beginning will be delivered by the deadline.
 * Sequential: High risk. Any delay in any phase can result in overshooting the deadline with nothing to deliver.
 
 c) Total project cost.
 
-* Iterative:  We can always stop before the project budget is exceeded. However, this does not guarantee that all features promised at the beginning will be delivered under the estimated cost. (The sequential model requires us to carry on even if the budget is exceeded because there is no intermediate version to fall back on).
+* Iterative: We can always stop before the project budget is exceeded. However, this does not guarantee that all features promised at the beginning will be delivered under the estimated cost. (The sequential model requires us to carry on even if the budget is exceeded because there is no intermediate version to fall back on).
```

Comment:
```suggestion
* Iterative: We can always stop before the project budget is exceeded. However, this does not guarantee that all features promised at the beginning will be delivered under the estimated cost. 
* Sequential: The sequential model requires us to carry on even if the budget is exceeded because there is no intermediate version to fall back on.
```
I suggest changing the layout to this for consistency with the rest of the answers.